Panic if oper-classes title information is left blank

My config:

```yaml
oper-classes:
   "foo":
     title:
```
error:
```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/ergochat/ergo/irc.(*Config).OperatorClasses(0xc0001ba400, 0xc0002b6300, 0xc, 0xc00023c8b8)
        /go/src/github.com/ergochat/ergo/irc/config.go:757 +0xaf6
github.com/ergochat/ergo/irc.LoadConfig(0xc000178177, 0x9, 0x9c4ee4, 0x6, 0xc00014b048)
        /go/src/github.com/ergochat/ergo/irc/config.go:1425 +0xe5e
main.main()
        /go/src/github.com/ergochat/ergo/ergo.go:146 +0x7c5
```

problem is the `[0]` in the title here:

https://github.com/ergochat/ergo/blob/stable/irc/config.go#L757

--- a/irc/config.go
+++ b/irc/config.go
@@ -747,6 +747,9 @@ func (conf *Config) OperatorClasses() (map[string]*OperClass, error) {
 
 			// add our own info
 			oc.Title = info.Title
+			if oc.Title == "" {
+				oc.Title = "IRC operator"
+			}
 			for _, capab := range info.Capabilities {
 				oc.Capabilities.Add(fixupCapability(capab))
 			}

